jvm/src/main/scala/org/portablescala/reflect/InstantiatableClass.scala

```diff
@@ -1,26 +1,35 @@
 package org.portablescala.reflect
 
-/** A wrapper for a class that can be instantiated.
+import java.lang.reflect.InvocationTargetException
+
+/**
+ * A wrapper for a class that can be instantiated.
  *
  *  @param runtimeClass
  *    The `java.lang.Class[_]` representing the class.
  */
 final class InstantiatableClass private[reflect] (val runtimeClass: Class[_]) {
+
   /** A list of the public constructors declared in this class. */
   val declaredConstructors: List[InvokableConstructor] =
-    runtimeClass.getConstructors().map(new InvokableConstructor(_)).toList
+    runtimeClass.getConstructors.map(new InvokableConstructor(_)).toList
 
-  /** Instantiates this class using its zero-argument constructor.
+  /**
+   * Instantiates this class using its zero-argument constructor.
    *
    *  @throws java.lang.InstantiationException
    *    (caused by a `NoSuchMethodException`)
    *    If this class does not have a public zero-argument constructor.
    */
   def newInstance(): Any = {
     try {
-      runtimeClass.newInstance()
+      runtimeClass.getDeclaredConstructor().newInstance()
```

Comment:
This is a welcome fix, but should go into a separate unit of change (commit or PR)

Reply:
Good idea! https://github.com/portable-scala/portable-scala-reflect/pull/40